app/views/layouts/application.html.erb

```diff
@@ -21,6 +21,34 @@
     </script>
     <%= javascript_include_tag "vendor/polyfill-output-value.js" %>
     <%= javascript_include_tag "vendor/outerHTML.js" %>
+
+    <% gtm_container = "GTM-M6GS7FF" %>
+    <% gtm_id = "G-1RH26G5KVP" %>
```

Comment:
I think we'll want to use different IDs depending on the environment. Could you extract this into a helper?